new-file

```diff
@@ -0,0 +1 @@
+Electric boogaloo
```

Comment:
```suggestion
yo this is cooked m8
```

### CHECK IT YO

<details>
  <summary>Click me</summary>
  
  ### Heading
  1. Foo
  2. Bar
     * Baz
     * Qux

  ### Some Javascript
  ```js
  function logSomething(something) {
    console.log('Something', something);
  }
  ```
</details>